src/libsystemd-network/dhcp-protocol.h

```diff
@@ -11,6 +11,11 @@
 
 #include "macro.h"
 #include "sparse-endian.h"
+#include "time-util.h"
+
+/* RFC 8925 - IPv6-Only Preferred Option for DHCPv4 3.4.
+ * MIN_V6ONLY_WAIT: The lower boundary for V6ONLY_WAIT. Value: 300 seconds */
+#define MIN_V6ONLY_WAIT_USEC (300 * USEC_PER_SEC)
```

Comment:
i'd suffic the 300 with `U`, so that we don't have to remember what signed mutliplied by unsigned results in. (i think unsigned, but i am not sure)